repos.d/rpm/rosa.yaml

```diff
@@ -74,7 +78,34 @@
       url: 'https://abf.io/import/{srcname}/tree/rosa2019.0'
   tags: [ all, production, rosa ]
 
-# Note: since Rosa 2019.1, use of urpmi metadate (info.xml) is discouraged
+# Note: since Rosa 2019.05, use of urpmi metadate (info.xml) is discouraged
+- name: rosa_2019_05
+  type: repository
+  desc: Rosa 2019.05
+  statsgroup: Rosa
+  family: rosa
+  ruleset: [rosa, rpm]
+  color: '3757a1'
+  minpackages: 6500
+  sources:
+    # - rosa2019.0 and rosa2019.05 are platforms for special purposes (certification), restricted is merged into
+    # main and is removed because it makes no sense there as US patent do not work in Russia which is the only
+    # region where certified distros will be distributed;
+    # - contrib does not exist there;
```

Comment:
`contrib/release` is non-empty for both 2019.0 and 2019.05, please elaborate

Reply:
thanks for noting this, they have been cleaned from some random packages